modules/ROOT/pages/dw-core-functions-flatten.adoc

```diff
@@ -8,15 +8,16 @@ endif::[]
 [[flatten1]]
 == flatten&#40;Array<Array<T&#62; &#124; Q&#62;&#41;: Array<T &#124; Q&#62;
 
-Flattens an array of arrays into a single, simple array.
+Turns a nested array (such as `[ [1], [2] ]`) into a simple array, such as
+`(such as `[1, 2]`)`.
```

Comment:
Did you mean (such as `[1, 2]`)?  the backticks are duplicated in the external ()

Reply:
thanks for noticing that, I'll fix